shared/src/business/useCases/trialSessions/batchDownloadTrialSessionInteractor.js

```diff
@@ -162,6 +180,17 @@ const batchDownloadTrialSessionInteractor = async (
     });
   };
 
+  const onError = error => {
+    applicationContext.getNotificationGateway().sendNotificationToUser({
```

Comment:
should we also log this to the error logs? Otherwise it's just the user who will see this error message?

Reply:
Yeah, I actually removed that by accident...
https://github.com/ustaxcourt/ef-cms/pull/1070/files#diff-3e2d7a71712966189525afdc55445ca74523cc3da42c233a36eebf1f62756fe2L73

Reply:
(well, technically I removed it on purpose to test something, but forgot to put it back)